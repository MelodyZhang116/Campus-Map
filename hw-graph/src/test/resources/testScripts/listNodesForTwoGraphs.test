# test that nodes (from two graphs) can be listed correctly. test whether the nodes in
# one graph listed can have effect on other graph.

# Create a graph
CreateGraph graph1

# Add a node to graph1
AddNode graph1 n1

# Add a node to graph1
AddNode graph1 n2

# List the nodes in the graph1
ListNodes graph1

# Create another graph
CreateGraph graph2

# Add a node to the second graph
AddNode graph2 n3

# Add a node to the second graph
AddNode graph2 n4

# List the nodes in the second graph
ListNodes graph2

# List the nodes in the first graph again to make sure it is still correct
ListNodes graph1