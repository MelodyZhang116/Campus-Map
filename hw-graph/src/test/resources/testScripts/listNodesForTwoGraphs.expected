# test that nodes (from two graphs) can be listed correctly. test whether the nodes in
# one graph listed can have effect on other graph.

# Create a graph
created graph graph1

# Add a node to graph1
added node n1 to graph1

# Add a node to graph1
added node n2 to graph1

# List the nodes in the graph1
graph1 contains: n1 n2

# Create another graph
created graph graph2

# Add a node to the second graph
added node n3 to graph2

# Add a node to the second graph
added node n4 to graph2

# List the nodes in the second graph
graph2 contains: n3 n4

# List the nodes in the first graph again to make sure it is still correct
graph1 contains: n1 n2
